language/move-ir-compiler/src/unit_tests/function_tests.rs

```diff
@@ -87,25 +49,3 @@ fn compile_module_with_large_frame() {
     let compiled_module_res = compile_module_string(&code);
     assert!(compiled_module_res.is_ok());
 }
-
-#[test]
-fn compile_module_with_script_visibility_functions() {
-    let code = String::from(
-        "
-        module 0xa1.Foobar {
-            public(script) foo() {
-            label b0:
-                return;
-            }
-
-            public(script) bar() {
-            label b0:
-                Self.foo();
-                return;
-            }
-        }
-        ",
-    );
-    let compiled_module_res = compile_module_string(&code);
-    assert!(compiled_module_res.is_ok());
-}
```

Comment:
These tests have moved to `language/move-ir-compiler/transactional-tests/tests/bytecode-generation/declarations/function.mvir`.